### ISSUE TITLE
Progrss doesnt get reset even if I cancel (ctrl+c) while using winget to install any package

### ISSUE THREAD

**Issue by its-ag** [microsoft/winget-cli#3431]
### Brief description of your issue

![image](https://github.com/microsoft/winget-cli/assets/102473837/dcf170c7-aac1-45cb-9507-ab2e9d5c9867)

The progress bar didnt get removed/cleared even if I cancel a winget installation. 

![image](https://github.com/microsoft/winget-cli/assets/102473837/e1d21f5d-c37b-4739-96aa-c56e8e39bd02)

Same for the taskbar.

### Steps to reproduce

- try to install any package.
e.g; `` winget install Neovim.Neovim``
- press ctrl+c to stop the installation (halfway of installing)
- Notice the windoes terminal progress bar (and in top left of the tab in windows teminal)

### Expected behavior

It should reset those green progress bars too.

### Actual behavior

![image](https://github.com/microsoft/winget-cli/assets/102473837/3967f057-4c53-48d2-ad74-d4eb8fd6b858)
This is stopped halfway even after cancellation where it should just ge reset. (as it was befoee installing anything)

### Environment

```shell
[winget --info]
Windows Package Manager v1.5.1572
Copyright (c) Microsoft Corporation. All rights reserved.

Windows: Windows.Desktop v10.0.19045.3086
System Architecture: X64
Package: Microsoft.DesktopAppInstaller v1.20.1572.0
```

**Comment by denelon** [comment]
@DHowett I was hoping I could enlist your help on this one. I'm guessing there may be a VT sequence or something that needs to be sent to reset the state for Windows Terminal and we might not be doing that when we receive [Ctrl]+[c].

**Comment by DHowett** [comment]
Absolutely. If you have a custom handler for <kbd>Ctrl+C</kbd>, you can use it to emit the same control sequence that you already use when you clear the progress on completion.

IF you do not have a handler for <kbd>Ctrl+C</kbd>, you won't be able to clear the progress (as is tradition; applications exiting ungracefully always leave terminals in weird states.)
